app/view_models/forgot_password_show.rb

```diff
@@ -0,0 +1,26 @@
+class ForgotPasswordShow
+  def initialize(params:, session:)
```

Comment:
I like that the view model "knows less", it's just dealing with the direct values. For example, why does the view model mutate the session currently? Seems weird to me.

However I don't think we have an established pattern here so I'm also fine leaving it as is.